scripts/redis/setup_redis_acl.sh

```
@@ -62,56 +62,62 @@ done
-echo "User ACL setup completed successfully. Lock file created at $LOCK_FILE"
+# Check if the ACL file already exists
+if [ -f "$acl_file" ]; then
+    echo "Redis ACL file already exists. Skipping creation."
+else
+    # Ensure the ACL file exists and set proper permissions
+    touch "$acl_file"
+    chmod 644 "$acl_file"
+    echo "Redis ACL file created and permissions set."
+
+    # Write the ACL content to the file
+    echo "user $username1 on >$password1 ~* +@all" > "$acl_file"
```

Shellcheck (0.10.0)

[warning] username1 is referenced but not assigned (did you mean 'username'?).

(SC2154)